themes/startwords/theme.toml

```diff
@@ -1,12 +1,12 @@
-name = "Blank"
+name = "startwords"
 license = "MIT"
-licenselink = "https://github.com/vimux/blank/blob/master/LICENSE"
-description = "Starter Hugo theme for developers."
+licenselink = "https://github.com/Princeton-CDH/startwords/blob/master/themes/startwords/LICENSE"
+description = "Theme for the Startwords research periodical."
 homepage = "https://github.com/vimux/blank/"
-tags = ["blog", "plain", "blank", "starter", "development"]
+tags = ["journal", "research", "periodical"]
```

Comment:
Thanks for updating these!

Reply:
It looks like we should update the license, it still lists vimux as author.  

The theme license is MIT, can we / do we want to change that?

I don't think we actually have a license on the codebase yet, but we should probably add one.

Reply:
let's remove the theme license (since we don't intend to distribute it independently). maybe we can use Apache2 for the main codebase, since we use that on most other projects.